### PR TITLE
Weartest wear tokotlin

### PR DESCRIPTION
**Pull request by nicolbidone**

### CODE REVIEW

File: mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java [hunk 1]
```diff
@@ -83,14 +83,16 @@ public void sendNotification(ConfigurationActivity configurationActivity) {
...
+        NotificationCompat.Action accion_abrir = new NotificationCompat.Action.Builder(android.R.drawable.ic_menu_send, "Volver App", pendingIntent).build();
         NotificationCompat.Builder notificationBuilder = new NotificationCompat.Builder(configurationActivity);
         notificationBuilder.setSmallIcon(R.drawable.ic_launcher);
         notificationBuilder.setContentTitle(NOTIFICATION_TITLE);
         notificationBuilder.setContentText(NOTIFICATION_TEXT);
         notificationBuilder.setContentIntent(pendingIntent);
         notificationBuilder.setAutoCancel(true);
-
+        notificationBuilder.setStyle(new NotificationCompat.BigTextStyle()
+                .bigText("La aplicación recibio una nueva notificacion , decidi la accion de seguir en esta activity o pasar a la 2da activity."))
```
Comment (mauroLonghi) [inline mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java]: hardcode

File: mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationActivity.java [hunk 2]
```diff
@@ -56,7 +56,7 @@ protected void onDestroy() {
     }
 
 //    @OnClick(R.id.button_change_background)
-    void onClickChangeBackground(View view) {
+    public void onClickChangeBackground(View view) {
         mConfigurationPresenter.changeContentImage(isConnected, Constants.BACKGROUND_CHOOSER);
         //mConfigurationPresenter.sendNotification(this);
```
Comment (mauroLonghi) [inline mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationActivity.java]: comments

File: wear/src/main/java/com/yalantis/watchface/service/AbstractAnalogWatchFaceService.kt [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+         *
+         * @param canvas          Canvas instance on draw ticks on.
+         * @param minutesRotation Angle of minute tick rotation.
+         * @param hoursRotation   Angle of hour tick rotation.
+         */
+        private fun drawAmbientTicks(canvas: Canvas, minutesRotation: Float, hoursRotation: Float) {
+            val mFilterPaint = Paint()
+            mFilterPaint.isFilterBitmap = true
+            mFilterPaint.isAntiAlias = false
+            val scale_modifier = 10
```
Comment (mauroLonghi) [inline wear/src/main/java/com/yalantis/watchface/service/AbstractAnalogWatchFaceService.kt:374]: aca tmb

File: wear/src/main/java/com/yalantis/watchface/service/WearableListenerService.kt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import com.google.android.gms.wearable.MessageEvent
+import com.yalantis.watchface.events.WatchfaceUpdatedEvent
+import de.greenrobot.event.EventBus
+
+class WearableListenerService : com.google.android.gms.wearable.WearableListenerService() {
+
+    override fun onMessageReceived(messageEvent: MessageEvent) {
+        Log.i("WearableListenerService","messageReceived")
+
+        val offset = 0
```
Comment (mauroLonghi) [inline wear/src/main/java/com/yalantis/watchface/service/WearableListenerService.kt:14]: idem

File: wear/src/main/java/com/yalantis/watchface/service/AbstractAnalogWatchFaceService.kt (hunk 3 above)
Comment (nicolbidone) [reply wear/src/main/java/com/yalantis/watchface/service/AbstractAnalogWatchFaceService.kt:374]: Done

File: wear/src/main/java/com/yalantis/watchface/service/WearableListenerService.kt (hunk 4 above)
Comment (nicolbidone) [reply wear/src/main/java/com/yalantis/watchface/service/WearableListenerService.kt:14]: Done

File: mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationActivity.java (hunk 2 above)
Comment (nicolbidone) [reply mobile/src/main/java/com/yalantis/watchface/view/configuration/ConfigurationActivity.java]: Done

File: mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java (hunk 1 above)
Comment (nicolbidone) [reply mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java]: Done

**Pull request merged**